OmniBLE/PumpManagerUI/ViewControllers/OmnipodSettingsViewController.swift

```diff
@@ -317,188 +316,188 @@ class OmnipodSettingsViewController: UITableViewController {
         }
     }
     
```

Comment:
Here I just uncommented this method as this is needed to prevent a crash when the table view requests a UITableViewCell.